README.md

```diff
@@ -3,9 +3,7 @@
 An image processing library for moving object detection implemented with GPUs.  
 Based on a Maximum Likelihood detection algorithm for moving astronomical objects.
 
-[![Build Status](https://github.com/dirac-institute/kbmod/actions/workflows/test_build.yaml/badge.svg)](https://github.com/dirac-institute/kbmod/actions/workflows/test_build.yaml)[![Documentation](https://github.com/dirac-institute/kbmod/actions/workflows/build_docs.yaml/badge.svg)](https://epyc.astro.washington.edu/~kbmod/) [![License](https://img.shields.io/badge/License-BSD%202--Clause-orange.svg)](https://opensource.org/licenses/BSD-2-Clause)[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1342297.svg)](https://doi.org/10.5281/zenodo.1342297)
-
-
+[![Build Status](https://github.com/dirac-institute/kbmod/actions/workflows/test_build.yaml/badge.svg)](https://github.com/dirac-institute/kbmod/actions/workflows/test_build.yaml)[![Documentation](https://github.com/dirac-institute/kbmod/actions/workflows/build_docs.yaml/badge.svg)](https://epyc.astro.washington.edu/~kbmod/) [![License](https://img.shields.io/badge/License-BSD%202--Clause-orange.svg)](https://opensource.org/licenses/BSD-2-Clause)[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7666852.svg)](https://doi.org/10.5281/zenodo.7666852)
```

Comment:
We might not want to change this Zenodo link. The original link will always send the user to the latest version on Zenodo. I think that might be better than having to always remember to update this link.

Reply:
Sounds good. I didn't realize that this was already a "latest version" link. Better to keep it that way.